transfer.md

```diff
@@ -0,0 +1,49 @@
+# cri-o Usage Transfer
+
+This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes cri-o.
+
+## Operational Transfer
+
+## Abstract
+
+The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use (See the document for [configuring kubernetes to use cri-o](./kubernetes.md) for more information on that).
```

Comment:
suggest "... Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document...)" or a few other words of your choosing to explain how Kubernetes uses the daemon.  Or perhaps just change "Kubernetes to use" to "Kubernetes operations".